src/backend/core/migrations/0002_remove_team_slug.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.1.1 on 2024-10-23 15:19
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
The migration should be delayed to a further release to avoid errors in production during the deployment.
We didn't implement syzygy yet!